packit_service/worker/build/build_helper.py

```diff
@@ -373,9 +389,28 @@ def _report(
         The status reporting should be done through this method
         so we can extend it in subclasses easily.
         """
-        self.status_reporter.report(
-            description=description, state=state, url=url, check_names=check_names,
-        )
+        try:
+            self.status_reporter.report(
+                description=description, state=state, url=url, check_names=check_names,
+            )
+        except Exception as e:
+            logger.info("Unable to report status ", e)
+            self.report_status_by_comment(
+                int(self.metadata.identifier), state, url, check_names
+            )
+
+    # Reporting build status if no permission to the fork project
```

Comment:
Use docstring for that information. (The `"""something"""` string bellow the function/class header.)